### PR TITLE
Add countryconfig service to SMS token audience

### PR DESCRIPTION
**Pull request by euanmillar**
Also needs this to enable SMS: https://github.com/opencrvs/opencrvs-farajaland/pull/453

### CODE REVIEW

File: packages/auth/src/features/verifyCode/service.ts [hunk 1]
```diff
@@ -74,8 +74,9 @@ export async function sendVerificationCode(
       Authorization: `Bearer ${await createToken(
         'auth',
         ['service'],
-        ['opencrvs:notification-user'],
-        JWT_ISSUER
+        ['opencrvs:notification-user', 'opencrvs:countryconfig-user'],
+        JWT_ISSUER,
+        true
```
Comment (euanmillar) [inline packages/auth/src/features/verifyCode/service.ts:79]: Adding true means that this JWT is only valid for 10 minutes.  It should have been there before

Comment (Zangetsu101) [inline packages/auth/src/features/verifyCode/service.ts:79]: This boolean means that the token is for a system. That's not the case here no?

**Review by Zangetsu101 — COMMENTED** [review]
@euanmillar There is also this place: https://github.com/opencrvs/opencrvs-core/blob/develop/packages/auth/src/features/retrievalSteps/sendUserName/service.ts#L27

File: packages/auth/src/features/verifyCode/service.ts (hunk 1 above)
Comment (Zangetsu101) [reply packages/auth/src/features/verifyCode/service.ts:79]: Ah okay so it makes the token temporary. The name is confusing tbh

Comment (euanmillar) [reply packages/auth/src/features/verifyCode/service.ts:79]: Its not the case, no, but if we didnt have this, then this JWT is essentially valid for a month.  Theres a small risk of abuse of the SMS gateway.

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**